deploy/monitor/push_prometheus_rule.yaml

```diff
@@ -14,78 +14,33 @@ spec:
           summary: "The UnifiedPush Pod Server is down. For more information see on the UnifiedPush at https://github.com/aerogear/aerogear-unifiedpush-server"
-          sop_url: "https://github.com/aerogear/unifiedpush-operator/blob/0.1.0/SOP/SOP-push.adoc" #todo
+          sop_url: "https://github.com/aerogear/unifiedpush-operator/blob/0.1.0/SOP/SOP-push.adoc"
       - alert: UnifiedPushConsoleDown
-        expr: absent(kube_endpoint_address_available{endpoint="example-unifiedpushserver-unifiedpush"} == 1)
+        expr: absent(kube_endpoint_address_available{endpoint="example-unifiedpushserver-unifiedpush"} < 1)
         for: 5m
         labels:
           severity: critical
         annotations:
           description: "The UnifiedPush admin console has been down for more than 5 minutes. "
           summary: "The UnifiedPush admin console endpoint has been unavailable for more that 5 minutes. For more information see on the UnifiedPush  at https://github.com/aerogear/UnifiedPush "
-          sop_url: "https://github.com/aerogear/unifiedpush-operator/blob/0.1.0/SOP/SOP-push.adoc" #todo
+          sop_url: "https://github.com/aerogear/unifiedpush-operator/blob/0.1.0/SOP/SOP-push.adoc"
       - alert: UnifiedPushDatabaseDown
-        expr: absent(kube_pod_container_status_running{namespace="unifiedpush",container="postgresql"}==1)
+        expr: absent(kube_pod_container_status_running{namespace="unifiedpush",container="postgresql"} < 1)
         for: 5m
         labels: 
           severity: critical
         annotations:
           description: "The UnifiedPush Database pod has been down for more than 5 minutes"
           summary: "The UnifiedPush Database is down. For more information see on the UnifiedPush at https://github.com/aerogear/aerogear-unifiedpush-serve"
-          sop_url: "https://github.com/aerogear/unifiedpush-operator/blob/0.1.0/SOP/SOP-push.adoc" #todo
-      - alert: UnifiedPushPodCPUHigh # todo check if it will work after expse the metrcis
-        expr: "(rate(process_cpu_seconds_total{job='example-unifiedpushserver-unifiedpush'}[1m])) > (((kube_pod_container_resource_limits_cpu_cores{namespace='mobile-security-service',container='application'})/100)*90)"
-        for: 5m
-        labels:
-          severity: warning
-        annotations:
-          description: "The UnifiedPush Server Pod has been at 90% CPU usage for more than 5 minutes"
-          summary: "The UnifiedPush Server Pod is reporting high cpu usage for more that 5 minutes. For more information see on the UnifiedPush  at https://github.com/aerogear/UnifiedPush "
-          sop_url: "https://github.com/aerogear/unifiedpush-operator/blob/0.1.0/SOP/SOP-push.adoc" #todo
-      - alert: UnifiedPushPodMemoryHigh # todo check if it will work after expse the metrcis
-        expr: "(process_resident_memory_bytes{job='example-unifiedpushserver-unifiedpush'}) > (((kube_pod_container_resource_limits_memory_bytes{namespace='unifiedpush',container='ups'})/100)*90)"
```

Comment:
@wei-lee and @grdryn the same above scenario here for memory.

Reply:
Let's leave the minimal required now and if be required we can add it in the future